lms/djangoapps/courseware/views.py

```diff
@@ -1065,7 +1066,9 @@ def _progress(request, course_key, student_id):
     # checking certificate generation configuration
     show_generate_cert_btn = certs_api.cert_generation_enabled(course_key)
 
-    if is_credit_course(course_key):
+    credit_course_requirements = None
+    is_course_credit = settings.FEATURES.get("ENABLE_CREDIT_ELIGIBILITY", False) and is_credit_course(course_key)
+    if is_course_credit:
```

Comment:
@jimabramson this is the change which causes SQL queries to drop. As here feature flag is checked first and on false return ```is_credit_course(course_key)``` is not called and hence corresponding query does not execute.